packages/vue/src/components/search/DataSearch.jsx

```diff
@@ -785,6 +811,10 @@ const DataSearch = {
 			this.triggerClickAnalytics(suggestion._click_id);
 			if (value === undefined) {
 				this.setValue(suggestion.value, true, this.$props, causes.SUGGESTION_SELECT);
+				this.isSuggestionSelected = true;
+				setTimeout(() => {
+					this.isSuggestionSelected = false;
+				}, 50);
```

Comment:
This fixes a side effect I am observing with two calls being made to this.setValue otherwise. See the use of the variable for the second place where the second call is being made from.